ProfileView/README.md

```diff
@@ -0,0 +1,79 @@
+# CJaaS Profile View Widdget
+
+This widget uses CJaaS Common Components to output a combined customer profile view that includes Profile information an ACtivity Timeline side by side.
+
+
+## Profile Widget Properties
+
+The CJaaS Profile Widget accepts specific properties to interact with the CJaaS API
+* _customer: string_ -  an identifier of the customer, i.e. "560000-John"
+* _template: any | JSON | Object_  -  for user-provided data-shape template.
+* _auth-token: string_ -  an unique Auth token to enable the CJaaS tape stream.
+* _base-url: string_ -  defaults to "https://trycjaas.exp.bz" at present, can be modified for changing APIs
+
+```html
+<cjaas-profile-view-widget
+   customer="560000-John"
+   .template=${sampleTemplateObject}
+   authToken="st=demoassure&............=sandbox&sig=qnKHkG1aAZryxbBfgTLG1XR8jLFbztQ4xKyn5APjdSY="
```

Comment:
if this is an actual token, let's not add it to the readme, rather describe where people can get it

Reply:
it's just a dummy token-like strin=g,  not an actual